recipe/meta.yaml

```
@@ -24,6 +29,8 @@ requirements:
     # numpy.distutils has a complex mechanism to facilitate mixing gfortran and MSVC
     # https://pav.iki.fi/blog/2017-10-08/pywingfortran.html#building-python-wheels-with-fortran-for-windows
     - {{ compiler('m2w64_fortran') }}   # [win]
+    # pythran-build needs clang-cl on windows
+    - clang                             # [win]
```

You could try using the activation for `clang` on Windows to have all the environment variables correctly setup. 
```suggestion
    - {{ compiler('clang') }}                             # [win]
```

---

Thanks, didn't know that the activation isn't done fully by `clang` itself already...